src/OrchardCore.Modules/OrchardCore.AuditTrail/Views/Items/AuditTrailAdminFilters-Category.Thumbnail.cshtml

```diff
@@ -4,11 +4,10 @@
 }
 
 <h4 class="card-title">@T["Category"]</h4>
-
-<pre class="mb-3">@(@term != null ? "category:" + @term.ToString() : "category:...")</pre>
+<pre class="mb-3">@(term?.ToString() ?? "category:...")</pre>
```

Comment:
Why "category:..." is not localized?

Reply:
It's the term name to use concretely (like a doc), it's like a method name, for example in the docs we don't localize the name of liquid filters